include/deal.II/lac/trilinos_block_sparse_matrix.h

```diff
@@ -563,7 +563,78 @@ namespace TrilinosWrappers
       BaseClass::vmult_nonblock_nonblock (dst, src);
   }
 
-}
+  namespace internal
+  {
+    namespace BlockLinearOperator
+    {
+
+      /**
+      * This is a extension class to BlockLinearOperators for Trilinos
+      * block sparse matrices.
```

Comment:
And also, what exactly does this class really do? It has a bunch of constructors, but it has no member variables, and none of the constructors either do, or save, something. I think I haven't quite understood the payload concept yet :-)